### PR TITLE
issue 50 resolved - new html reference link and remove event listener…

### PR DESCRIPTION
**Pull request by cepps646**
… button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,12 +15,19 @@
...
 <body bgcolor="#3393FF">
         <h1>Whats The Weather Today?</h1>
-        <div class="Navagation">
+        <!--<div class="Navagation">
 
             <input id="atlanta" type="button" class="button" onclick="indexAtlanta.html" value="Go to Atlanta Hub" />
             
             <input id="dallas" type="button" class="button" onclick="indexDallas.html" value="Go to Dallas Hub" />
-        </div>
+        </div> -->
```
Comment (davidgrieser) [inline index.html:23]: Commented out HTML can be removed.

**Comment by cepps646** [comment]
image

**Comment by cepps646** [comment]
updated appropriately. Thanks.

**Review by davidgrieser — COMMENTED** [review]
Have some commented out HTML that can be removed. But otherwise this works.

Screen Shot 2021-08-12 at 1 32 19 PM

**Comment by cepps646** [comment]
appropriately removed comments. Thanks. 
image

**Review by davidgrieser — APPROVED** [review]
That is a clean PR (Pull Request)

LGTM (Looks Good To Merge)

**Comment by davidgrieser** [comment]
This resolves #50

**Pull request merged**